.github/workflows/containerd.yml

```diff
@@ -0,0 +1,91 @@
+name: Containerd CI
+on: [push, pull_request]
+
+jobs:
+  #
+  # Run CRI tests against containerd
+  #
+  linux-build-and-test-containerd:
+    strategy:
+      matrix:
+        version: [master, v1.4.0-beta.1]
+    name: critest - containerd ${{matrix.version}} (linux amd64)
```

Comment:
a little wordy to say Containerd CI/critest -containerd/

Reply:
suggest "containerd ${{matrix.version}} arch"

Reply:
because of the matrix options this will get pretty long..

Reply:
or you could make the top name just containerd.. and mention critest for this name.. either way works..

Reply:
Ok, changed it. Lets see what you think.